### ISSUE TITLE
TypeError: e is null

### ISSUE THREAD

**Issue by ThwCorbin** [PrismJS/prism#2105]
**Information:**
- Prism version: PrismJS 1.17.1
- Plugins: none
- Environment: Node, 11ty static site generator, Rollup bundler, Netlify CDN

**Description**
Prism is working fine, although I get a type error when pages load. Here is the type error:
- In Chrome: `Uncaught TypeError: Cannot read property 'defer' of null`
- In Firefox: `TypeError: e is null`

Here is the code the type error references:

```
    var e = _.util.currentScript();

    if (e && (_.filename = e.src, e.hasAttribute("data-manual") && (_.manual = !0)), !_.manual) {
      var n = function n() {
        _.manual || _.highlightAll();
      };

      var t = document.readyState;
      "loading" === t || "interactive" === t && e.defer ? document.addEventListener("DOMContentLoaded", n) : window.requestAnimationFrame ? window.requestAnimationFrame(n) : window.setTimeout(n, 16);
    }
```
I am not using prismjs and babel-plugin-prismjs packages, as directed in the Prism docs "Usage with Webpack, Browserify, & Other Bundlers" as I cannot get this setup to work with my bundler, Rollup. Instead, I've pasted the prism.js code into the bottom of my script.js, which Rollup bundles.

**Example**
This is the website where I have the bug: https://www.thwcorbin.com
And here is the code in my repository: https://github.com/ThwCorbin/toms-site/blob/master/src/assets/js/script.js

**Comment by RunDevelopment** [comment]
Thank you for reporting!

I made a fix for the issue (I caused; truely sorry), so the fix will be available as soon as #2106 is merged.

**Comment by ThwCorbin** [comment]
Thank you!
Cheers,
Tom
